CHANGELOG.md

```diff
@@ -3,6 +3,58 @@
+- fix: Address misc. bugs in create-component script ([#232](https://github.com/Workday/canvas-kit/pull/232))
+- fix: Change casing for aria-labelledby attr ([#236](https://github.com/Workday/canvas-kit/pull/236))
+- fix: Move uuid from defaultProps to component instance ([#228](https://github.com/Workday/canvas-kit/pull/228))
+- ci: Create github action for CI and add Cypress to CI ([#240](https://github.com/Workday/canvas-kit/pull/240))
+
+### Components:
+
+- fix(checkbox): Change default id to be unique per instance ([#192](https://github.com/Workday/canvas-kit/pull/192))
+- fix(menu): Add Menu example with icons and grow support ([#147](https://github.com/Workday/canvas-kit/pull/147))
+- fix(button): Fix accessibility styling for CSS buttons ([#186](https://github.com/Workday/canvas-kit/pull/186))
+- fix(core): Update incorrect alert colors
+- fix(menu): Always focus selected menu item and add tab accessibility ([#239](https://github.com/Workday/canvas-kit/pull/239))
+
+### Breaking Changes:
+
+- fix(core): Update error colors to pass a11y
+- refactor(tooltip): Update CSS styles to match React implementation ([#49](https://github.com/Workday/canvas-kit/pull/49))
+- refactor(forms): Split CSS forms module into individual modules ([#24](https://github.com/Workday/canvas-kit/pull/24))
+- refactor(loading-animation): Update CSS styles to match React implementation ([#83](https://github.com/Workday/canvas-kit/pull/83))
+- refactor(card): Update CSS styles to match React implementation ([#113](https://github.com/Workday/canvas-kit/pull/113))
+- refactor(menu): Update CSS styles to match React implementation ([#117](https://github.com/Workday/canvas-kit/pull/117))
+- refactor(button): Update CSS styles to match React implementation ([#53](https://github.com/Workday/canvas-kit/pull/53))
+- fix(button): CSS button fixes ([#156](https://github.com/Workday/canvas-kit/pull/156))
+- refactor(page-header): Refactor CSS to match React implementation ([#151](https://github.com/Workday/canvas-kit/pull/151))
+- refactor(icon): Update CSS styles to match React implementation ([#159](https://github.com/Workday/canvas-kit/pull/159))
+- refactor(layout): Update CSS naming to match React implementation ([#168](https://github.com/Workday/canvas-kit/pull/168))
+- refactor(popup): Refactor CSS to match React implementation ([#155](https://github.com/Workday/canvas-kit/pull/155))
+- fix(popup): CSS cleanup ([#188](https://github.com/Workday/canvas-kit/pull/188))
+- refactor(form): Update CSS form styles and structure to match React implementation ([#112](https://github.com/Workday/canvas-kit/pull/112))
+- fix(button): Use correct variable name after rebase error
+- fix: Update snapshots and fix popup stories after rebase
+- refactor(tooltip): Update CSS story to match the React version ([#198](https://github.com/Workday/canvas-kit/pull/198))
+- fix(button): Update incorrect delete button colors
+- fix(common): Use correct alert color variable for common errors
```

Comment:
There are 3 lines dedicated to this change as we moved things around. Since none are tied to a PR, let's consolidate them into one breaking change.